### PR TITLE
fix: inherit notifiers.smtp from root smtp configuration

### PR DESCRIPTION
**Pull request by kamiiiel**
## Issue

https://gravitee.atlassian.net/browse/APIM-1394

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-pfwaeptjoy.chromatic.com)

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: apim
 # Also update CHANGELOG.md
-version: 4.0.0
+version: 4.0.1
 appVersion: 4.0.0
```
Comment (julian-robotham-gravitee) [inline helm/Chart.yaml]: @kamiiiel Does the app version need to change?

Comment (leleueri) [reply helm/Chart.yaml]: Don't think so, this appVersion represent the default value used to pull the images when the chart is deployed. So 4.0.0 looks good to me as we are on the master branch.

File: helm/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
 
 This file documents all notable changes to [Gravitee.io API Management 3.x](https://github.com/gravitee-io/helm-charts/tree/master/apim/3.x) Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+### 4.0.1
+- 
+- Inherit notifiers.smtp from root smtp configuration
```
Comment (leleueri) [inline helm/CHANGELOG.md]: Should we highlight that it is a breaking change?

File: helm/Chart.yaml [hunk 3]
```diff
@@ -19,10 +19,4 @@ annotations:
...
   # https://artifacthub.io/packages/helm/graviteeio/apim?modal=changelog
   artifacthub.io/changes: |
-    - Change the versioning
-    - Remove duplicate annotation in ui deployment
-    - Add support for managed Service Account for each product
-    - Remove old and unused `cache.type` from gateway config map
-    - Fix resources missing in ui-deployment.yaml
-    - Change readinessProbe for the gateway to sync-process
-    - Update Elasticsearch version
+    - Inherit notifiers.smtp from root smtp configuration
```
Comment (leleueri) [inline helm/Chart.yaml:31]: Should we highlight that it is a breaking change?

**Review by leleueri — COMMENTED** [review]
Wondering if the target branch should be 4.0.x instead and use the "apply-to-master" label to cherry-pick this change on master

File: helm/Chart.yaml (hunk 1 above)
Comment (kamiiiel) [reply helm/Chart.yaml]: So we don't change the chart version?

File: helm/CHANGELOG.md (hunk 2 above)
Comment (kamiiiel) [reply helm/CHANGELOG.md]: I have to add this change on the 4.0.0 too?

**Review by sebdevaux — APPROVED** [review]

**Pull request merged**